IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json

```diff
@@ -1296,22 +1292,22 @@
 						],
 						"body": {
 							"mode": "raw",
-							"raw": "{ \n\t\"entity\" :\n\t{\n\t\t\"Type\" : {\"value\" : \"Invoice\"}, \n\t    \"ReferenceNbr\" : {\"value\" : \"000088\"} \n\t},\n\t\"parameters\" : \n\t{}\n}"
+							"raw": "{ \n\t\"entity\" :\n\t{\n\t\t\"OrderType\" : {\"value\" : \"SO\"}, \n\t    \"OrderNbr\" : {\"value\" : \"000001\"} \n\t},\n\t\"parameters\" : \n\t{}\n}"
 						},
 						"url": {
-							"raw": "{{EndpointURL}}SalesInvoice/ReleaseSalesInvoice",
+							"raw": "{{EndpointURL}}SalesOrder/ReopenSalesOrder",
 							"host": [
-								"{{EndpointURL}}SalesInvoice"
+								"{{EndpointURL}}SalesOrder"
 							],
 							"path": [
-								"ReleaseSalesInvoice"
+								"ReopenSalesOrder"
```

Comment:
Why ReopenSalesOrder here if we have release of a sales order invoice in the description?